Tasks/AzureRmWebAppDeployment/task.json

```diff
@@ -94,7 +94,7 @@
             "label": "Package or Folder",
             "defaultValue": "",
             "required": true,
```

Comment:
Can we add default value to package input as $(System.DefaultWorkingDirectory)\**\*.zip.